### PR TITLE
Add faces detection example

### PR DESCRIPTION
**Pull request by SimakovSergey**
## Description

Adding the new example for computer vision - faces detection. It is focused on a specific task of detecting faces with the bounding boxes together with indicating how to use it with polygons.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation and examples improvement (changes affected documentation and/or examples)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by SimakovSergey** [comment]
@fed

File: examples/faces_detection/faces_detection.ipynb [hunk 1]
```diff
@@ -0,0 +1,987 @@
+{
```
Comment (DrhF) [inline examples/faces_detection/faces_detection.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    !pip install toloka-kit # To interact with Toloka API
</pre><p>Please. specify the version of toloka-kit (1.0.0 is current version)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (DrhF) [inline examples/faces_detection/faces_detection.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    !pip install crowd-kit
</pre><p>Please, specify the version of crowd-kit (1.1.0 is current version)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (DrhF) [inline examples/faces_detection/faces_detection.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #20.        (toloka.filter.Skill(detection_skill.id) != None)
</pre><p>It is stated in a comment above that users, who performed detection tasks, shouldn't be allowed to perform verification tasks. But if worker performed detection tasks, their detection skill is not None. So we need to revert the inequality to equality operator: </p><pre class="ql-syntax" spellcheck="false">Line #20.        (toloka.filter.Skill(detection_skill.id) == None)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

**Review by DrhF — APPROVED** [review]

**Pull request merged**